docs: :memo: update iteration.qmd (#227)
seedcase-project/team: how-we-work/iteration.qmd

    iterations a few days before major holidays start.
-   Will plan and decide on how long the future iterations should be
    (can range from 2-6 weeks, but prefer shorter iterations over longer

>     ones). Currently, we find iterations with a duration of one month

Reviewer: ```suggestion
    ones). Currently, we find iterations with a duration of one calendar month
```

::: callout-note
A basic agenda for the meeting might be:

-   Review and agree on the iteration aims and end date.
-   Review the list of tasks already listed.
-   Brainstorm and add any other issues as needed.
-   Schedule the next iteration's end and start meetings.

We usually have separate aims for each person in the team, and each of

> us will add tasks to the board relevant to that aim aim as needed. This

Reviewer: ```suggestion
us will add tasks to the team project board relevant to that aim aim as needed. This
```
Would be my suggestion, I can't see that the board is introduced above, but I may have missed it.

tasks.

Then, we agree on and schedule the end date debrief and retrospective
meeting, which should be approximately 1-2 hours and preferably before

> lunch, as well as the next iterations planning meeting. The next
> planning meeting should either be on the same date after lunch, to
> ensure a proper break before moving to a new iteration, or the next day

Reviewer: ```suggestion
lunch, as well as the next iterations planning meeting. We have found that the
planning meeting is best held the day after to ensure that we don't get too tired towards the end. Alternatively it can be the same day as the debrief, but there should be a substantial break between the two meetings.
```
